src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(value, currantResult) {
+      return currantResult + value;
+    },
+    subtract(value, currantResult) {
```

Comment:
The parameter `currantResult` is misspelled. Consider changing it to `currentResult`.